### PR TITLE
chore: update to react@16.9

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

This depends on the dependency update (#963) as otherwise we end up with two react versions again.

Then tests would run and we could see where we are.

This does not update the peer depenedency but I don't think we have to stricly speaking?

### CODE REVIEW

**Comment by emmenko** [comment]
> This does not update the peer depenedency but I don't think we have to stricly speaking?

Let's see, maybe it's not necessary

**Comment by tdeekens** [comment]
The trick here is to keep `release/15` up to date with `master` and this up to date with the release branch so we have one version of react. Otherwise tests fail.

File: package.json [hunk 1]
```diff
@@ -144,7 +144,8 @@
     "typescript": "3.5.3"
   },
   "resolutions": {
-    "@types/react": "16.9.1"
+    "@types/react": "16.9.1",
+    "react": "16.9.0"
```
Comment (tdeekens) [inline package.json]: Just to experiment. Without continue to end up with two react versions due to `react-intl`.

**Comment by tdeekens** [comment]
I think now the question may be how much of those warnings come from our libs and should we silence their logging as discussed.

**Comment by tdeekens** [comment]
Done. You want to do the rebase of the apollo branch?

**Comment by emmenko** [comment]
Sure I can do that later. Thanks

**Pull request merged**